docs/sphinx/user_guide/getting_started.rst

```diff
@@ -0,0 +1,76 @@
+.. _getting_started-label:
+
+*************************
+Getting Started with CAMP
+*************************
+
+This page provides information on how to quickly get up and running with Camp
+
+------------
+Installation
+------------
+
+Camp is hosted on GitHub `here <https://github.com/LLNL/Camp>`_.
+To clone the repo into your local working space, type:
+
+.. code-block:: bash
+
+  $ git clone --recursive https://github.com/LLNL/Camp.git
+
+
+The ``--recursive`` argument is required to ensure that the *BLT* submodule is
+also checked out. `BLT <https://github.com/LLNL/BLT>`_ is the build system we
+use for Camp.
+
+
+^^^^^^^^^^^^^
+Building Camp
+^^^^^^^^^^^^^
+
+Camp uses CMake and BLT to handle builds. Make sure that you have a modern
+compiler loaded and the configuration is as simple as:
+
+.. code-block:: bash
+
+  $ mkdir build && cd build
+  $ cmake ../
+
+By default, Camp will only support the Host backend. Additional backends for
+device support can be enabled using the options such as ``-DENABLE_CUDA=On``
+CMake will provide output about which compiler
+is being used and the values of other options. Once CMake has completed, Camp
+can be built with Make:
+
+.. code-block:: bash
+
+  $ make
+
+-----------
+Basic Usage
+-----------
+
+Most Camp usage revolves around Camp resources. A quick way to get started
```

Comment:
```suggestion
Most Camp usage revolves around Camp ``Resource`` types. A quick way to get started
```

When introducing key concepts in Camp, I think this kind of emphasis helps readers know that something is a Camp type/class, etc.